### DISCUSSION TITLE
Add back the ability to reorder query parameters

### DISCUSSION THREAD

**Discussion by Wazbat** [Kong/insomnia#5766] [Ideas] [answered]
While the new query parameter page does look nice, and the ability to add descriptions could be useful, it appears the ability to reorder query params has been lost.

If I'm not mistkane, you used to be able to drag the left side of each row to reorder, and now this functionality has been lost

**Comment by mrstevek** [comment]
Reordering parameters was very useful.  Now the app is missing a high-value feature.  Please bring it back as soon as possible.  Even if it needs a configuration change to enable it, it's worth it to have the option.

**Comment by hrvoj3e** [comment]
This is an essential feature.
Please enable it back again.

**Comment by webocoders** [comment]
Please enable it back again.

**Comment by digiacom** [comment]
I would also appreciate this feature !

**Comment by subnetmarco** [comment]
We will re-enable it.

**Reply by kleenkanteen** [reply]
Has this been enabled? And do update insomnia do I have to download and reinstall it from the website? I can't find a update button.

**Reply by subnetmarco** [reply]
This will ship in v9.3 soon.

**Comment by subnetmarco** [comment, marked as answer]
This [has been released](https://konghq.com/blog/product-releases/insomnia-9-3-ga) in Insomnia 9.3
